debian/tests/integrationtests

```diff
@@ -16,14 +18,25 @@ fi
 # ensure our PATH is right
 . /etc/profile.d/apps-bin-path.sh
 
-tmp="$(mktemp -d)"
+tmp="${ADT_ARTIFACTS}/build"
+mkdir -p $tmp
 export GOPATH=$tmp
-./get-deps.sh
 mkdir -p $GOPATH/src/github.com/ubuntu-core/
-cp -R . $GOPATH/src/github.com/ubuntu-core/snappy/
-mkdir $GOPATH/src/github.com/ubuntu-core/snappy/integration-tests/data/output
+cp -R . $GOPATH/src/github.com/ubuntu-core/snappy/ && rm -rf $GOPATH/src/github.com/ubuntu-core/snappy/.git
```

Comment:
There are no hidden files (.*) that we depend upon for building. Instead of copying everything and then removing, which is double work, can we instead change the first command to be `cp -R * $GOPATH/...etc..`?